test/indexer/__init__.py

```diff
@@ -136,3 +136,31 @@ def mocked_get_bundle(bundle_uuid, bundle_version):
             with patch.object(indexer, '_get_bundle', new=mocked_get_bundle):
                 contributions = indexer.transform(notification, delete=False)
                 return indexer.contribute(contributions)
+
+    def _verify_sorted_lists(self, value, path):
+        """
+        Traverse through 'value' to verify all lists of primitives are sorted
```

Comment:
Explain what sort of JSON this function accepts (index documents and service responses).